examples/src/dzi/dzi-viewer.tsx

```diff
@@ -63,6 +56,7 @@ export function DziView(props: Props) {
         }
         return () => {
             if (cnvs.current) {
+                // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I think this is out of place? This isn't disabling the `useEffect` dependency lint issues. We also still need to actually integrate a linter soon 😅